code/modules/vending/drinnerware.dm

```diff
@@ -13,7 +13,7 @@
 		            /obj/item/reagent_containers/food/condiment/pack/hotsauce = 5,
 		            /obj/item/reagent_containers/food/condiment/saltshaker = 5,
 		            /obj/item/reagent_containers/food/condiment/peppermill = 5,
-		            /obj/item/reagent_containers/glass/bowl = 20)
+		            /obj/item/reagent_containers/glass/bowl = 50)
```

Comment:
Make it return the empty bowls. Don't do this because you don't know how.

Reply:
You don't start with enough rice to use all of these bowls to begin with, nevermind the fact they should be re-usable.

Reply:
I legit tryed. I tryed lists, I tryed making them be the trash item. Im just to cat brained to make it work

Reply:
If you want to make a way for crafting to become list friendly and able to work. That would be grand but as it is, best thing I can do is make bowls more easy to make. Let alone the fact that the dish drive treats them as trash.
Ideally the Dish Drive makes new dishes rather then just teles them into trash cans